examples/export-data/utils/types.ts

```diff
@@ -0,0 +1,424 @@
+import {
```

Comment:
don't copy this over - instead import any types you need from the bog-analytics npm package - delete this file